2022/pt-br/Days/day05.md

```diff
@@ -43,43 +43,42 @@ Uma vez implantado, vamos operá-lo. E operá-lo pode envolver algo como você c
 
-## Rinse & Repeat
+## Limpe & Repita
 
-Once that's in place you go right back to the beginning to the planning stage and go through the whole thing again
+Uma vez implementado, você volta ao início, ao estágio de planejamento, e repassa tudo novamente.
 
-## Continuous
+## Contínua
 
-Many tools help us achieve the above continuous process, all this code and the ultimate goal of being completely automated, cloud infrastructure or any environment is often described as Continuous Integration/ Continuous Delivery/Continous Deployment or “CI/CD” for short. We will spend a whole week on CI/CD later on in the 90 Days with some examples and walkthroughs to grasp the fundamentals.
+Muitas ferramentas nos ajudam a alcançar o processo contínuo acima, todo esse código e o objetivo final de ser completamente automatizado, a infraestrutura em nuvem ou qualquer ambiente é frequentemente descrito como Integração Contínua/Entrega Contínua/Implantação Contínua ou “CI/CD” para abreviar. Passaremos uma semana inteira sobre CI/CD mais tarde nos 90 dias com alguns exemplos e orientações para compreender os fundamentos.
 
-### Continuous Delivery
+### Entrega Contínua
 
-Continuous Delivery = Plan > Code > Build > Test
+Entrega Contínua = Planejar > Codar > Construir > Testar
 
-### Continuous Integration
+### Integração Contínua
 
-This is effectively the outcome of the Continuous Delivery phases above plus the outcome of the Release phase. This is the case for both failure and success but this is fed back into continuous delivery or moved to Continuous Deployment.
+Este é efetivamente o resultado das fases de Entrega Contínua acima, mais o resultado da fase de Lançamento(Release). Esse é o caso tanto do fracasso quanto do sucesso, mas isso é realimentado na entrega contínua ou movido para a implantação contínua.
 
-Continuous Integration = Plan > Code > Build > Test > Release
+Integração Contínua = Plano > Código > Construir > Teste > Lançamento(Release)
 
-### Continuous Deployment
+### Implantação Contínua
 
-If you have a successful release from your continuous integration then move to Continuous Deployment which brings in the following phases
+Se você tiver uma versão bem-sucedida de sua integração contínua, mude para a implantação contínua, que traz as seguintes fases
 
-CI Release is Success = Continuous Deployment = Deploy > Operate > Monitor
+O lançamento(release) da Integração Contínua foi bem-sucedida = Implantação Contínua = Implantar > Operar > Monitorar
 
-You can see these three Continuous notions above as the simple collection of phases of the DevOps Lifecycle.
+Você pode ver essas três noções Contínuas acima como uma simples coleção de fases do ciclo de vida do DevOps.
```

Comment:
Capitalize the term _DevOps Lifecycle_:
"Você pode ver essas três noções contínuas acima como a simples coleção de fases do Ciclo de Vida DevOps."